session_0/word_processing.rb

```diff
@@ -34,11 +35,45 @@ def words_with_prefix(prefix, words)
 # The function `similarity_score` takes two words and returns the
 # similarity score (an integer).
 def similarity_score(word_1, word_2)
-  raise NotImplementedError # TODO
+  size = [word_1.size,word_2.size].min # TODO
+  score = 0
+  for i in 0..size-1
```

Comment:
See if you can use `find` function here: https://apidock.com/ruby/Enumerable/find